### ISSUE TITLE
strolling down through comments very fast breaks the UI

### ISSUE THREAD

**Issue by fooditron** [yuliskov/SmartTubeNext#2701]
If you pull up the comments on any video, and then hold down the down-arrow button to scroll through comments quickly, the UI breaks and it becomes impossible to do anything other than hit the back-button to exit out of the comments.

**Comment by gate1975mlm** [comment]
Same here hope this can be fixed.

**Comment by eng3** [comment]
same here

**Comment by LearninBurnin** [comment]
**[Semi-workaround]** if the last comment (the comment that the ui becomes frozen on) has replies on the comment, you can hit ok button and view replies, and when you press back button to exit, you will be returned to your same place in the comment stream and it should be unfrozen as well. 

So if many of the comments have replies you can be a little bit riskier with your scroll speed lol. But you have to resume scrolling down at a painstakingly slow pace to continue through the chat If you run into a long stretch of comments with no replies. Any comment you read could be your last!

Being able to scroll the comments is a super sweet feature, I hope the kinks get worked out as soon as time allows!

**Comment by fooditron** [comment]
Just an update, this issue still exists :(

**Comment by eng3** [comment]
Yep.  Not surprised since I haven't seen any updates.  I don't think anyone has even looked at this issue yet considering its not assigned or labeled.
It's pretty useless when trying to view a past live stream since it gets shown as comments. You will usually have tons so you have to scroll.  And it's pretty hard to correlate to the video by time.

**Comment by eng3** [comment]
Well, the main reason I'd scroll down fast is because it froze and I was trying to return to the spot I was at.

The other reason was to try to keep keep up with a live stream manually but it never really worked.
Real support for live streams is still missing.
